### ISSUE TITLE
Using this package with R 4.3.0

### ISSUE THREAD

**Issue by apzentral** [CDCgov/Rnssp#16]
To make this package work with R version 4.3.0, we would also need to force the installation of the deprecate dependencies below,

```r
#install the Rnssp package
install.packages("devtools")
library(devtools)
install_bitbucket("richierocks/assertive.base")
install_bitbucket("richierocks/assertive.properties")
install_bitbucket("richierocks/assertive.types")
install.packages(c("magrittr", "purrr", "dplyr"))
devtools::install_github("cdcgov/Rnssp")
```

**Comment by rosericazondekon** [comment]
Thank you @apzentral for your contribution. Yes, your workaround will successfully install `Rnssp`. 

For your information, we've already fixed the dependency issue related to `assertive.types` and are planning a patch release soon.

In the meantime, you can install `Rnssp` from its dev branch with no issue by running the following in your R console:

```
devtools::install_github("cdcgov/Rnssp@dev")
```

**Comment by apzentral** [comment]
Perfect, thank you so much @rosericazondekon . I will update my colleague that we can also use the dev branch.

**Comment by rosericazondekon** [comment]
Dear @apzentral ,

We just released `Rnssp v0.3.0` which among other features, updates, and , resolves the `assertive.types` dependency issue. 

Please, feel free to check the new release note for an exhaustive list of the novelties.

You may install the newly released version by running:
```
devtools::install_github("cdcgov/Rnssp")
```

Thank you!
